frontend/web/micro-ui-internals/packages/ui-components/src/atoms/Banner.js

```
@@ -0,0 +1,108 @@
+      <header style={props?.props?.headerStyles ? props?.props?.headerStyles : {}}>{props.props.message}</header>
+      <SVG.Error />
+      <h2 style={props?.props?.infoStyles ? props?.props?.infoStyles : {}}>
+        {props?.props?.complaintNumber ? t("CS_PGR_COMPLAINT_NUMBER") : props.props.info}
+      </h2>
+      <p style={props?.props?.applicationNumberStyles ? props?.props?.applicationNumberStyles : {}}>
+        {props?.props?.complaintNumber ? props?.props?.complaintNumber : props?.props?.applicationNumber}
+      </p>
+      {props?.props?.multipleResponseIDs && (
+        <div className="banner-multiple-ids-wrapper">
+          {props?.props?.multipleResponseIDs.map((responseIDs) => (
+            <p>{responseIDs}</p>
```

Biome (2.1.2)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)